readme.md

```diff
@@ -158,7 +177,8 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:**
+I actually did not get an error.
```

Comment:
There _should_ be an error when running `.create(...).select(...)`. So... interesting?